.gitignore

```diff
@@ -1,4 +1,4 @@
 .p4*
 .DS_Store
 .AppleDouble
-
+*.pyc
```

Comment:
Since you're adding this - don't suppose you could add `__pycache__` to the list as well?
Had that queued up in another PR, but if you just add that to your MR, would avoid a merge conflict...

Reply:
Hi @chadrik, is the .gitignore change critical for the main goal of this PR? For tracking changes for USD releases, it helps if all the changes for a PR are related. 

If the .gitignore change isn't specifically needed for this PR, would it be possible to move the .gitignore change (maybe combined with @pmolodo's change) to a separate PR?

Reply:
Hi @chadrik, is the .gitignore change critical for the main goal of this PR? For tracking changes for USD releases, it helps if all the changes for a PR are related.

If the .gitignore change isn't specifically needed for this PR, would it be possible to move the .gitignore change (maybe combined with @pmolodo's change) to a separate PR?

Reply:
Removed!